testing/btest/Baseline/opt.opt-no-files/.stderr

```diff
@@ -1,2 +1,2 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
-fatal error: no matching functions/files for C++ compilation
+fatal error: no matching functions/files for script optimization
```

Comment:
As of tests, it might be useful to have a new test where these assignment chains are explicitly constructed. There's some in the base scripts, but some more direct usage might good for coverage, comparing -O ZAM vs non-optimized runs.

Reply:
Yeah, good thought. I've added a BTest and confirmed that ZAM generates the optimized code for it (and still passes :-).